cpu/kinetis/periph/gpio.c

```diff
@@ -148,10 +148,13 @@ static inline int pin_num(gpio_t pin)
 
 /**
  * @brief   Get context for a specific pin
+ *
+ * @return  index of context for specified pin
+ * @return  -1 if pin has no context
  */
 static inline int get_ctx(int port, int pin)
 {
-    return (isr_map[(port * 4) + (pin >> 3)] >> ((pin & 0x7) * 4)) & 0xf;
+    return ((isr_map[(port * 4) + (pin >> 3)] >> ((pin & 0x7) * 4)) & 0xf) - 1;
```

Comment:
Should this also be done in `write_map`?

Reply:
The equivalent is done in the only consumer of `write_map()` that needs it, in `gpio_init_int()`